.travis.yml

```diff
@@ -20,17 +20,19 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
 before_install:
-  - curl -L "https://apt.llvm.org/llvm-snapshot.gpg.key" | sudo apt-key add -
-  - curl -L "http://packages.lunarg.com/lunarg-signing-key-pub.asc" | sudo apt-key add -
-  - echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main" | sudo tee -a ${TRAVIS_ROOT}/etc/apt/sources.list
-  - echo "deb https://packages.lunarg.com/vulkan bionic main" | sudo tee -a ${TRAVIS_ROOT}/etc/apt/sources.list
-  - sudo apt-get update
-  - sudo apt-get -yq --no-install-suggests --no-install-recommends install
-      llvm-${LLVM_VERSION}-dev
-      clang-${LLVM_VERSION}
-      clang-format-${LLVM_VERSION}
-      clang-tidy-${LLVM_VERSION}
-      spirv-tools
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
```

Comment:
The newlines are getting swallowed by YAML, here.

Would probably recommend a literal block scalar as we use elsewhere:
```
  - |
    if [ $TRAVIS_OS_NAME == "line" ]; then
```
(... or you could add semicolons, instead.)

Reply:
Good catch!